smooth/framework/functions/update_annuities.py

```diff
@@ -25,6 +31,9 @@ def update_annuities(component):
     if not component.op_emissions:
         op_emissions = 0
     else:
+        # Check if 'variable' op_emissions are beeing used, if so decide which op_emissions are valid

```

Comment:
Line is too long. Fix the spelling error (beeing -> being), then it should fit. Check with flake8.